### PR TITLE
Fix: Arb/OP watcher 

### PR DESCRIPTION
**Pull request by wsdt**
Based on migrate/graphql branch since history had several bugs that were already fixed in the prev PR. 

## Overview

Fixes 1 minor bug.

## Changes

- OmgWatcher for OP/ARB removed & made it optional in general, since not all networks may have it.

## Testing

Local testing.

### CODE REVIEW

File: src/components/global/dropdown/index.tsx
Comment (jemeza) [inline src/components/global/dropdown/index.tsx:1]: Let's not make this change yet, it may break other things. I'll put up a fix for this shortly

**Review by jemeza — CHANGES_REQUESTED** [review]
nice

File: src/components/global/dropdown/index.tsx
Comment (wsdt) [reply src/components/global/dropdown/index.tsx:1]: sure reverted.

**Review by jemeza — APPROVED** [review]
Let's get this merged!

**Pull request merged**